### PR TITLE
Fix youtube creators, release 0.6.2

### PR DESCRIPTION
**Pull request by Powersource**

### CODE REVIEW

**Comment by Powersource** [comment]
We still get some undefined errors, but we poll so much that it works anyway. Should open a separate issue.

File: src/content_scripts/contentlib.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import browser from 'webextension-polyfill';
 
-async function queryElement(query, node) {
+function queryElement(query, node) {
```
Comment (ErikBjare) [inline src/content_scripts/contentlib.js]: Why did you remove async?

File: src/content_scripts/contentlib.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export async function waitForElements(
   let error;
   for (let i = 0; i < retries; i++) {
     try {
-      return await Promise.all(queries.map(query => queryElement(query, node)));
+      return queries.map(query => queryElement(query, node));
```
Comment (ErikBjare) [inline src/content_scripts/contentlib.js]: I don't get why you no longer await the promises?

File: src/content_scripts/contentlib.js (hunk 1 above)
Comment (Powersource) [reply src/content_scripts/contentlib.js]: Because the function doesn't do anything that's async. https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector doesn't return a promise.

File: src/content_scripts/contentlib.js (hunk 2 above)
Comment (Powersource) [reply src/content_scripts/contentlib.js]: Because they're not promises (anymore).

File: src/content_scripts/contentlib.js (hunk 1 above)
Comment (ErikBjare) [reply src/content_scripts/contentlib.js]: Oh sorry, read the diff wrong. Makes sense.
